lib/ansible/modules/cloud/amazon/ec2_win_password.py

```diff
@@ -153,15 +158,12 @@ def main():
     else:
         try:
             with f:
-                key = RSA.importKey(f.read(), key_passphrase)
-        except (ValueError, IndexError, TypeError) as e:
+                key = load_pem_private_key(f.read(), key_passphrase, BACKEND)
+        except (ValueError, TypeError) as e:
             module.fail_json(msg = "unable to parse key file")
 
-    cipher = PKCS1_v1_5.new(key)
-    sentinel = 'password decryption failed!!!'
-
     try:
-        decrypted = cipher.decrypt(decoded, sentinel)
+        decrypted = key.decrypt(decoded, PKCS1v15())
     except ValueError as e:
         decrypted = None
 
```

Comment:
@s-hertel @ryansb this particular  change is isolated to the ec2_win_password.py  What do you think about it?  Should we add a documented cryptography dep for this module and apply this change or should we change it to do the right thing with either cryptography or pycrypto (and document the requirement for one or the other in the docs)?